Packs/Shodan/Integrations/Shodan_v2/Shodan_v2.py

```diff
@@ -209,21 +260,42 @@ def search_command():
                         "Latitude": location["latitude"],
                         "Country": location["country_name"],
                     },
-                    "Timestamp": match.get("timestamp", ""),
-                    "Domains": match.get("domains", []),
-                    "OS": match.get("os", ""),
+                    'Timestamp': match.get('timestamp', ''),
+                    'Domains': match.get('domains', []),
+                    'OS': match.get('os', ''),
+                    'Data': match.get('data', ''),
+                    'Hash': match.get('hash', 0)
```

Comment:
Please keep all in single or double quotes

Reply:
Changed my single quotes to double quotes to match style used throughout script.